11ty/definitions/algorithmic-opacity.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Algorithmic Opacity
+slug: algorithmic-opacity
+defined: true
+speech: noun
+---
+
+The effect of an algorithmic system designed in a way that makes difficult to explain why a certain decision was made by it.
+
+An algorithmic system may be kept opaque deliberately to conceal how it operates, unintentionally due to the expert technical knowledge required to understand it, or as result of a mismatch between how humans and algorithms understand the input given.
```

Comment:
```suggestion
## Issues
An algorithmic system may be kept opaque deliberately to conceal how it operates, unintentionally due to the expert technical knowledge required to understand it, or as result of a mismatch between how humans and algorithms understand the input given.
```
Is there the potential for intentional harm done by humans by way of the algorithm?